src/components/common/Spinner.tsx

```diff
@@ -0,0 +1,22 @@
+export const Spinner = () => (
+  <svg
+    className="h-4 w-4 animate-spin text-[#FF486E]"
+    xmlns="http://www.w3.org/2000/svg"
+    fill="none"
+    viewBox="0 0 24 24"
+  >
```

Comment:
Make it actually reusable and configurable, Then add `h-4 w-4 text-[#FF486E]` class name on usage.

```suggestion
import { ComponentProps } from "react";

import { cn } from "@/utils/ui";

export const Spinner = ({ className, ...props }: ComponentProps<"svg">) => (
  <svg
    className={cn("animate-spin", className)}
    xmlns="http://www.w3.org/2000/svg"
    fill="none"
    viewBox="0 0 24 24"
    {...props}
  >
```